frontend/public/components/edit-yaml.jsx

```diff
@@ -328,14 +328,12 @@ const EditYAMLInner = (props) => {
     (obj, model, newNamespace, newName) => {
       setSuccess(null);
       setErrors(null);
-      let action = k8sUpdate;
       let redirect = false;
-      if (create) {
-        action = k8sCreate;
-        delete obj.metadata.resourceVersion;
-        redirect = true;
-      }
-      action(model, obj, newNamespace, newName)
+      const response = create
+        ? ((redirect = true), delete obj.metadata.resourceVersion, k8sCreate(model, obj))
```

Comment:
not new, but we shouldn't mutate `obj`. Instead, we should use a copy, omitting the unwanted properties:
```suggestion
        ? k8sCreate(model, _.omit(obj, ['metadata', 'resourceVersion']))
```